test/connectSpec.js

```diff
@@ -110,6 +110,37 @@ describe('Connection methods', function () {
       infoNock.done()
     })
 
+    it('should set urls from object in metadata', function * () {
```

Comment:
Fixed in https://github.com/interledger/js-ilp-plugin-bells/pull/52/commits/71ee1b7e82db7817f3be32da6345943605cb7de8